frontend/src/components/__tests__/__snapshots__/VisualizeChart.test.tsx.snap

```diff
@@ -499,7 +499,7 @@ exports[`renders the VisualizeChart component 1`] = `
             >
               <div
                 class="recharts-responsive-container"
-                id="title"
+                id="line-chart-undefined"
```

Comment:
Is there a way to set the `widgetId` property for the `*ChartWidget` components rendered inside of `VisualizeChart` component to get rid of of the `undefined` value that is getting appended to their id attributes?

Reply:
I added an id of "visualization" here. There is no access to the widget id here because the widget may not have been created yet. The id with visualization is unique because you can only visualize one chart at a time.